lib/scanner/blocks.go

```diff
@@ -16,6 +16,24 @@ import (
 	"github.com/syncthing/syncthing/lib/sha256"
 )
 
+const (
+	// Shifts
+	KiB = 10
+	MiB = 20
+	GiB = 30
+)
+
+var blockSizes = []int{
+	128 << KiB, // up to 250 MiB
+	256 << KiB, // up to 500 MiB
+	512 << KiB, // up to 1000 MiB
+	1 << MiB,   // up to 2000 MiB
+	2 << MiB,   // up to 4000 MiB
+	4 << MiB,   // up to 8000 MiB
+	8 << MiB,   // up to 16000 MiB
+	16 << MiB,  // to infinity and beyond
+}
```

Comment:
A list of blocksizes is defined both here and in protocol without reference to the min and max blocksizes defined in protocol. In principle this list choice is arbitrary (e.g. rounding at 1MiB), so I think we should only be doing it in one place and refer to it in other places. And maybe to be super cautious, generate the min/max-blocksizes from this list.